README.md

```diff
@@ -1 +1,4 @@
-# Wargear
+# Scripts the help with playing risk like games
```

Comment:
Scripts ~the~ that/to help with playing ~risk like~ risk-like games

Reply:
Nice catch!